src/traces/contour/constants.js

```diff
@@ -72,5 +72,8 @@ module.exports = {
         INITIALSEARCHPOINTS: 10,
         // number of binary search iterations after the initial wide search
         ITERATIONS: 5
-    }
+    },
+
+    // number of px around labels text clip out of contour lines below
+    LABEL_LINE_CLIP_PAD: 2
```

Comment:
Could you please try 3px horizontal padding here?

Reply:
2 px looks like plenty to me, but I might argue for making it a fraction of the font size (or text height) so that it scales with the text.